subsys/emul/espi/emul_espi_host.c

```diff
@@ -128,10 +128,9 @@ static int emul_host_find_index(struct espi_host_emul_data *data,
 static int emul_host_set_vw(struct espi_emul *emul, enum espi_vwire_signal vw,
 			    uint8_t level)
 {
-	struct espi_host_emul_data *data;
+	struct espi_host_emul_data *data = emul->parent->data;
```

Comment:
Same here, can we pass the `struct emul *target`?

Reply:
Looks like I was lacking some changes in the headers as well as clients.

Done.